platforms/android/xwalk.gradle

```diff
@@ -25,6 +25,13 @@ def SHARED_ARTIFACT_ID = "xwalk_shared_library:"
 def EMBEDD_ARTIFACT_ID = "xwalk_core_library:"
 def CANARY_ARTIFACT_ID = "xwalk_core_library_canary:"
 def BIT_64 = ":64bit@aar"
+def MIN_SDK_VERSION = 16
+
+if (cdvMinSdkVersion == null) {
+    ext.cdvMinSdkVersion = MIN_SDK_VERSION;
+} else if (cdvMinSdkVersion.toInteger() < MIN_SDK_VERSION) {
+    ext.cdvMinSdkVersion = MIN_SDK_VERSION;
```

Comment:
It has been configured to 16 by default. Maybe it need to be announced if the cdvMinSdkVersion is modified lower to 16.

> if (cdvMinSdkVersion == null) {
    def minSdk = getConfigPreference("android-minSdkVersion");
    ext.cdvMinSdkVersion =  minSdk && minSdk.toInteger() > 14 ? minSdk : 14;
}